W-22512036 hyperforce 0630 duke (#480)
mulesoft/docs-monitoring: modules/ROOT/pages/log-points.adoc

= Generating Real-Time Logs
:page-info-pills: anypoint-pricing-titanium

> Generate logs for apps and APIs in real-time and without writing code. For apps, you can specify the level of logs you want to generate, and for APIs, you can specify properties for the logs. This allows you to extract logs on-demand from applications in real-time, without causing re-deployment or application downtime. Permissions control allows you to manage access. All logs seamlessly flow into log management.

Reviewer: ```suggestion
Generate logs for apps and APIs in real-time and without writing code. For apps, specify the level of logs to generate, and for APIs, specify properties for the logs. This allows you to extract logs on-demand from applications in real-time, without causing redeployment or app downtime. Manage access with permissions control. All logs flow in to log management.
```
